### PR TITLE
[Camden] Send photo URLs to field in Symology

### PR DESCRIPTION
**Pull request by chrismytton**
This field has been configured to accept FixMyStreet photo URLs and display them in the Symology interface.

This is largely based on the work @davea did in https://github.com/mysociety/open311-adapter/pull/190.

Fixes https://github.com/mysociety/societyworks/issues/3492

### CODE REVIEW

**Review by dracos — CHANGES_REQUESTED** [review]
Just little bit of documentation would be good :) Looks fine, can we chase Central Beds to get the same code there up and running, do we know what was happening there?

File: perllib/Open311/Endpoint/Integration/UK/Camden.pm [hunk 1]
```diff
@@ -15,4 +15,19 @@ has jurisdiction_id => (
 # Updates from FMS should always have a GN11 code, meaning "Customer called"
 sub event_action_event_type { 'NOTE' }
 
+sub process_service_request_args {
```
[the remark this answers is not in the document]
Comment (chrismytton) [reply perllib/Open311/Endpoint/Integration/UK/Camden.pm:25]: Added in 0588add

**Comment by chrismytton** [comment]
> can we chase Central Beds to get the same code there up and running, do we know what was happening there

Looks like Dave never got a response from his question last summer: https://mysocietysupport.freshdesk.com/a/tickets/1076

But yes, probably worth re-visiting the Central Beds code once this has been confirmed to work.

**Review by dracos — APPROVED** [review]

**Comment by chrismytton** [comment]
This is on the staging site now.

**Comment by MorayMySoc** [comment]
There is an update in what is being looked for - I'm not 100% this is the full solution as not sure if the 'attr' would need to change. Though potential global impact https://3.basecamp.com/4020879/buckets/27695641/todos/5740705094#__recording_5862342384

**Pull request merged**